frontend/src/index.css

```
@@ -1,6 +1,5 @@
-/* Fonts (Inter Variable for UI, Fraunces Variable for display headings) */
-@import '@fontsource-variable/inter';
-@import '@fontsource-variable/fraunces';
+/* Import Google Fonts */
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@400;500;700&family=Cuprum:wght@400;500;700&family=PT+Sans:wght@400;700&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Oswald:wght@400;500;700&family=Cuprum:wght@400;500;700&family=PT+Sans:wght@400;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Oswald:wght@400;500;700&family=Cuprum:wght@400;500;700&family=PT+Sans:wght@400;700&display=swap'" (import-notation)

(import-notation)